src/app/loadout-builder/processWorker/types.ts

```diff
@@ -0,0 +1,47 @@
+import { DestinyVersion } from '@destinyitemmanager/dim-api-types';
+import { DimSockets, DimStat } from '../../inventory/item-types';
+import { DestinyItemInstanceEnergy } from 'bungie-api-ts/destiny2';
+import { StatTypes } from '../types';
+
+export interface ProcessItem {
+  owner: string;
```

Comment:
Here apparently, https://github.com/DestinyItemManager/DIM/blob/master/src/app/loadout-builder/process.ts#L202

Reply:
Ah, that's just to default to the currently equipped ghost. I think I could be convinced to just drop ghosts now that they don't matter much anymore (guiding light or nothing) but this could also be replaced with a single boolean.

Reply:
I might drop ghosts for now as they don't get considered in the calculations. They can always be handled in an actual loadout too.